### PR TITLE
[NativeAOT-LLVM] Move dotnet.js to aotsdk

### PR DESCRIPTION
**Pull request by maraf**
Contributes to https://github.com/dotnet/runtimelab/issues/2434

### CODE REVIEW

**Comment by maraf** [comment]
cc @dotnet/nativeaot-llvm

File: eng/liveBuilds.targets [hunk 1]
```diff
@@ -229,15 +227,15 @@
...
+        $(CoreCLRAotSdkDir)dotnet.runtime.js;
+        $(CoreCLRAotSdkDir)dotnet.runtime.js.map;
+        $(CoreCLRAotSdkDir)dotnet.d.ts;
+        $(CoreCLRAotSdkDir)package.json;"
         IsNative="true" />
       <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
-                             Include="$(CoreCLRNativeArtifactsPath)src\es6\*.js"
+                             Include="$(CoreCLRAotSdkDir)src\es6\*.js"
                              NativeSubDirectory="src\es6"
                              IsNative="true" />
```
Comment (SingleAccretion) [inline eng/liveBuilds.targets]: This should not be needed anymore. Also, `Microsoft.DotNet.ILCompiler.LLVM.pkgproj` needs to be fixed to not copy these files into `framework/`.

File: src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets [hunk 2]
```diff
@@ -71,7 +71,7 @@
       <JSFileType Include="lib.js"     Kind="js-library" />
       <JSFileType Include="post.js"    Kind="post-js" />
       <JSFileType Include="extpost.js" Kind="extern-post-js" />
-      <_WasmExtraJSFile Include="$(IlcFrameworkNativePath)src\es6\*.%(JSFileType.Identity)" Kind="%(JSFileType.Kind)" />
+      <_WasmExtraJSFile Include="$(IlcSdkPath)src\es6\*.%(JSFileType.Identity)" Kind="%(JSFileType.Kind)" />
```
Comment (SingleAccretion) [inline src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets]: I suspect this needs tweaks to `Microsoft.DotNet.ILCompiler.LLVM.pkgproj` to preserve the `src/es6`. Alternatively, it needs fixes here to work with the flat hierarchy (which would be simpler if it's possible).

**Review by SingleAccretion — COMMENTED** [review]
Please also verify manually that the package still works and has the expected layout before merging.

File: src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets (hunk 2 above)
Comment (maraf) [reply src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets]: Good catch! I forgot changes to pkgproj in stash

**Comment by maraf** [comment]
image

**Review by SingleAccretion — APPROVED** [review]
Changes LGTM assuming the package works. Thank you!

**Pull request merged**